docs/reference/modules/shard-ops.asciidoc

```diff
@@ -1,7 +1,7 @@
 [[shard-allocation-relocation-recovery]]
 === Shard allocation, relocation, and recovery
 
-Each <<documents-indices,index>> in Elasticsearch is divided into one or more <<scalability,shards>>. 
```

Comment:
I'd rather this link stayed put.

Reply:
side benefit: apparently this doc is missing in 8.15 because I still don't understand branching. if you don't edit this link, the backport to 8.15 won't fail ;)